library/src/reactors/BasisTheoryReactors.ts

```diff
@@ -8,25 +12,25 @@ import type {
   UpdateReactorModel,
   ReactorQuery,
 } from './types';
-import {
-  transformReactorResponseCamelCase,
-  transformReactorRequestSnakeCase,
-} from './../common/utils';
 
 export const BasisTheoryReactors = new CrudBuilder(
   class BasisTheoryReactors extends BasisTheoryService {
     public constructor(options: BasisTheoryServiceOptions) {
-      options.transformRequest = ([] as AxiosTransformer[]).concat(
-        transformReactorRequestSnakeCase,
-        options.transformRequest || []
-      );
+      const _options = options;
```

Comment:
```suggestion
      const _options = {
        ...options
      };
```

Reply:
lol I suggested the opposite. why spread it here?

Reply:
So we don't do parameter reassign and don't just trick the linter.

Reply:
okay. NBD to me